src/picosvg/svg_types.py

```diff
@@ -512,10 +528,22 @@ def as_path(self) -> SVGPath:
 
 
 def union(shapes: Iterable[SVGShape]) -> SVGPath:
-    return SVGPath.from_commands(svg_pathops.union(*[s.as_cmd_seq() for s in shapes]))
+    return SVGPath.from_commands(
+        svg_pathops.union(
+            *[
+                s.as_path().remove_overlaps(is_clip_path=True).as_cmd_seq()
+                for s in shapes
+            ]
+        )
+    )
 
 
 def intersection(shapes: Iterable[SVGShape]) -> SVGPath:
     return SVGPath.from_commands(
-        svg_pathops.intersection(*[s.as_cmd_seq() for s in shapes])
+        svg_pathops.intersection(
+            *[
+                s.as_path().remove_overlaps(is_clip_path=True).as_cmd_seq()
+                for s in shapes
```

Comment:
suggest helper function for the preparation of args for a bool op; it's gotten elaborate enough to be worth